crates/levm/src/vm.rs

```diff
@@ -1295,6 +1307,10 @@ impl VM {
         self.call_frames.last().unwrap()
     }
 
+    pub fn add_account(&mut self, address: Address, account: Account) {
```

Comment:
In `Db` we have an `add_account` so we can drop this method

Reply:
Oops, it was left from some merge. Fixed!